sdk/identity/identity/src/credentials/onBehalfOfCredential.browser.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TokenCredential, AccessToken } from "@azure/core-auth";
+import { credentialLogger, formatError } from "../util/logging";
+
+const credentialName = "OnBehalfOfCredential";
+const BrowserNotSupportedError = new Error(`${credentialName}: Not supported in the browser.`);
+const logger = credentialLogger(credentialName);
+
+export class OnBehalfOfCredential implements TokenCredential {
```

Comment:
Aren't ClientSecret and ClientCert at least functional in browsers (even if that use case isn't supported?). Should OBO work in browsers as well if CSC does?

Reply:
Client Certificate Credential definitely doesn’t work in browsers: https://github.com/Azure/azure-sdk-for-js/blob/44712109c705d23d5ec52d98a3a508157674df84/sdk/identity/identity/src/credentials/clientCertificateCredential.browser.ts